beacon_node/lighthouse_network/gossipsub/src/config.rs

```diff
@@ -370,6 +371,16 @@ impl Config {
     pub fn forward_queue_duration(&self) -> Duration {
         self.connection_handler_forward_duration
     }
+
+    // Controls the message size threshold for which IDONTWANT messages are sent.
+    // Sending IDONTWANT messages for small messages can have a negative effect to the overall
+    // traffic and CPU load. This acts as a lower bound cutoff for the message size to which 
+    // IDONTWANT won't be sent to peers. Only works if the peers support Gossipsub1.3 
+    // (see https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.2.md#idontwant-message)
+    // default is 1kb
```

Comment:
Might be worth mentioning that the unit for the usize value is bytes.

Reply:
I always mess this up. resolved!